Common/Branding/BrandServers.swift

```diff
@@ -8,17 +8,23 @@ import Foundation
 public struct BrandServers: Decodable {
 
     private enum CodingKeys: CodingKey {
+        case dataPrivacy
+        case imprint
```

Comment:
Hm.. I don't really like putting these URLs right next to server endpoints. I would rather like to see two keys for `impirntURL` and `dataPrivacyURL` in the root of the brand config plist.